server/handlers.go

```diff
@@ -249,7 +249,7 @@ func (s *Server) handleAuthorization(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if len(connectors) == 1 {
+	if len(connectors) == 1 && !s.alwaysShowLogin {
```

Comment:
When there's only local users, I believe `len(connectors) == 1` also drives whether or not a "back link" is rendered:

![image](https://user-images.githubusercontent.com/870638/62462348-7bfa9b00-b787-11e9-97a4-2806c2efdd09.png)

I think that code needs adjustment, too, now? What would be desired if you click on "login with username/password" and then change your mind? Hm. If there's nothing else they can do, it might be ok to keep the logic as-is -- i.e. to not display the backlink regardless of `s.alwaysShowLogin`...

Reply:
I'd be in favor of keeping the behaviour the same as now